buildSrc/subprojects/kotlin-dsl/src/main/kotlin/plugins/kotlin-dsl-plugin-bundle.gradle.kts

```diff
@@ -110,9 +110,8 @@ afterEvaluate {
         isCanBeResolved = false
         isCanBeConsumed = true
         isVisible = false
-        outgoing.artifact(mapOf(
-            "file" to localRepository.get().asFile, // This is not lazy
-            "builtBy" to publishPluginsToTestRepository
-        ))
+        outgoing.artifact(localRepository) {
+            builtBy(publishPluginsToTestRepository)
```

Comment:
:o: Can't you just use `outgoing.artifact(publishPluginsToTestRepository)` now?

Reply:
See my answer below. Publishing tasks do not define outputs.